src/main/resources/lib/hudson/test/js/failureSummary.js

```diff
@@ -43,7 +43,10 @@ function handleShowHideClick(event) {
     const id = link.id.replace(/-showlink$/, '').replace(/-hidelink$/, '');
 
     if (link.id.endsWith('-showlink')) {
-        showFailureSummary(id, document.URL + id.replace(PREFIX, '') + "summary");
+        // clear the query parameters
+        const cleanUrl = new URL(document.URL);
+        cleanUrl.search = "";
```

Comment:
simpler:
```suggestion
        const cleanUrl = location.toString().replace(location.search, "");
```

Reply:
:shrug: Setting `.search` directly seems clearer and more reliable to me.

Reply:
It’s already clear enough that it needs a comment explaining it but sure we can go with this